static/js/components/inputs/FieldsOfStudySelectField.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+import SelectField from './SelectField';
+import FIELDS_OF_STUDY from '../../fields_of_study';
+
+let fieldOfStudyOptions = _.map(FIELDS_OF_STUDY, (name, code) => ({
+  value: code,
+  label: name
+}));
```

Comment:
here's a fun one. using `Object.entries` inside of the React component (or, most likely, anything that isn't executed immediately on startup) works fine, but using it outside of the class definition like this results in 'Object.entries is undefined'. the polyfill apparently hasn't had a chance to run yet when this line is executed